packages/ui/src/components/va-tree-view/components/VaTreeNode/VaTreeNode.vue

```diff
@@ -126,6 +126,10 @@ export default defineComponent({
       indent: props.node.hasChildren === false,
     }))
 
+    const cursorClassComputed = useBem('va-tree-node-content', () => ({
+      cursor: (props.node.hasChildren === true && expandNodeBy === 'node'),
```

Comment:
Basically, following BEM convention, you need to specify modifier. Let's have a specific cursor for a specific (expandable) content node.

```suggestion
      expandable: (props.node.hasChildren === true && expandNodeBy === 'node'),
```

Reply:
I changed it in the code, but i personally think that just `expandable` is a bit misleading because it only gets applied when `expandNodeBy` is also `"node"`, so maybe `expandablebynode` / `expandableByNode` (however this seems kind of repeating)

Reply:
Any Ideas?

Reply:
`on-expandable`.

Reply:
Make sense. Then it can be renamed to `clickable` or `interactive`.